src/Utils/Responsive/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { themeProps } from "@artsy/palette"
+import React from "react"
+import {
+  createResponsiveComponents,
+  ResponsiveProviderProps as _ResponsiveProviderProps,
+} from "./Responsive"
+
+type MediaQuery = keyof typeof themeProps["mediaQueries"]
+
+const ResponsiveComponents = createResponsiveComponents<MediaQuery>()
+
+export const Responsive = ResponsiveComponents.Consumer
+
+// TODO Once we consider the deprecation period of the previous ‘beakpoint’
+//      centric API to be over, we can replace the wrapper with just this line.
```

Comment:
We should add to this that we should refactor all the `NewResponsiveProviderProps` references to `ResponsiveProviderProps` after the deprecation is over. Also, if it would be good to go ahead and get issues created in the relevant projects so we can do the migration sooner rather than later. We shouldn't leave this hanging around.

Reply:
https://github.com/artsy/volt/issues/3116

Reply:
I don't think we really need to document that `NewResponsiveProviderProps` needs removing from the code base, because once all of this is removed the compiler will automatically point this out.

Reply:
Ahhh.... that's true, ha. Point taken.